scripts/SREP/hard-prune-image-registry/metadata.yaml

```diff
@@ -0,0 +1,146 @@
+      - pods
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - ""
+      resources:
+      - pods/exec
+      verbs:
+      - get
+      - create
+    - apiGroups:
+      - apps
+      resources:
+      - deployments
+      verbs:
+      - get
+  clusterRoleRules:
+    - apiGroups:
+      - imageregistry.operator.openshift.io
+      resources:
+      - configs
+      verbs:
+      - get
+      - patch
+    - apiGroups:
+      - rbac.authorization.k8s.io
+      resources:
+      - clusterrolebindings
+      verbs:
+      - create
+      - delete
+      - list
+    ## Required for system:image-pruner
+    - apiGroups:                                                
+      - ""                                                     
+      resources:                                                
+      - pods                                                    
+      - replicationcontrollers                                  
+      verbs:                                                    
+      - get                                                     
+      - list                                                    
+    - apiGroups:                                                
+      - ""                                                      
+      resources:                                                
+      - limitranges                                             
+      verbs:                                                    
+      - list                                                    
+    - apiGroups:                                                
+      - ""   
+      - build.openshift.io
+      resources:
+      - buildconfigs
+      - builds
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - ""
+      - apps.openshift.io
+      resources:
+      - deploymentconfigs
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - batch
+      resources:
+      - jobs
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - batch
+      resources:
+      - cronjobs
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - apps
+      resources:
+      - daemonsets
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - apps
+      resources:
+      - deployments
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - apps
+      resources:
+      - replicasets
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - apps
+      resources:
+      - statefulsets
+      verbs:
+      - get
+      - list
+    - apiGroups:
+      - ""
+      - image.openshift.io
+      resources:
+      - images
+      verbs:
+      - delete
+    - apiGroups:
+      - ""
+      - image.openshift.io
+      resources:
+      - images
+      - imagestreams
+      verbs:
+      - get
+      - list
+      - watch
+    - apiGroups:
+      - ""
+      - image.openshift.io
+      resources:
+      - imagestreams/status
+      verbs:
+      - update
```

Comment:
```suggestion
    - apiGroups:
      - rbac.authorization.k8s.io
      resources:
      - clusterrolebindings
      verbs:
      - bind
      resourceNames:
      - system:image-pruner
```

Reply:
Man, I wish I'd known I could do that yesterday!  :D  Thanks!